dedupe/api.py

```diff
@@ -845,6 +848,13 @@ def _loadSample(self, data_sample) :
                                                      self.data_model,
                                                      self.num_cores)
 
+    def _load_sampled_records(self, data_sample, sample_size=900):
+        if data_sample:
+            recs = itertools.chain.from_iterable(data_sample)
```

Comment:
This will work for Dedupe class but not for RecordLink classes, where we will need to set `self.sampled_records_1` and `self.sampled_records_2`